crates/core_arch/src/arm/neon/mod.rs

```diff
@@ -1368,6 +1368,21 @@ pub unsafe fn vgetq_lane_u32(v: uint32x4_t, imm5: i32) -> u32 {
     simd_extract(v, imm5)
 }
 
+/// Move vector element to general-purpose register
+#[inline]
+#[target_feature(enable = "neon")]
+#[cfg_attr(target_arch = "arm", target_feature(enable = "v7"))]
+#[rustc_args_required_const(1)]
+#[cfg_attr(all(test, target_arch = "arm"), assert_instr("vmov.32", imm5 = 2))]
+#[cfg_attr(all(test, target_arch = "aarch64"), assert_instr(mov, imm5 = 2))]
+pub unsafe fn vgetq_lane_s32(v: int32x4_t, imm5: i32) -> i32 {
+    if (imm5) < 0 || (imm5) > 3 {
+        unreachable_unchecked()
```

Comment:
Use a `panic` here instead. This will be optimized out since `imm5` is required to be a constant.